scripts/analytics/user_summary.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+import logging
+from datetime import datetime, timedelta
+from dateutil.parser import parse
+
+from modularodm import Q
+
+from website.app import init_app
+from website.models import User
+from website.settings import KEEN as keen_settings
+from keen.client import KeenClient
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def get_events(date):
+    """Count how many nodes exist.
+    If no date is given, include all users up until the point when the script was called.
```

Comment:
Looking at the current state of the code, `date` will never be not given.  Maybe delete this comment and add an assertion?